EIPS/eip-7704.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: 7704
```

Comment:
```suggestion
eip: 7709
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
Thank you, updated.